Test fitting (#139)
qiboteam/qibocal: doc/source/api-reference/qibocal.rst

.. autoclass:: qibocal.data.Data
    :members:
    :member-order: bysource

Fitting functions
-----------------

> ``Qibocal`` offers routine-specific method for post-processing acquired data with the :ref:`calibration_routines`.

Reviewer: ```suggestion
``Qibocal`` offers routine-specific method for post-processing analysis of the data generated by the different :ref:`calibration_routines`.

```
